WP_Auth0.php

```diff
@@ -31,43 +31,12 @@
 
 define( 'WPA0_LANG', 'wp-auth0' ); // deprecated; do not use for translations
 
-require_once WPA0_PLUGIN_DIR . 'vendor/autoload.php';
-require_once WPA0_PLUGIN_DIR . 'functions.php';
+require_once __DIR__ . '/vendor/autoload.php';
```

Comment:
Let's leave the autoloader require line as it is. `WPA0_PLUGIN_DIR` uses WordPress-y path calculation that should remain.

Reply:
`WPA0_PLUGIN_DIR`'s value comes from a WP core function that is not testable with static analysisi

`__DIR__` was not available in WordPress for a long time because of PHP 5.2 support!

Reply:
Fixed, we badly miss `composer validate --strict`

Reply:
Ref: https://stackoverflow.com/a/52221472/728480

Thanks for explanation!